outline.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Serverless perspectives
+subtitle: An outline 
+...
+
+I.  What is serverless?
+    A.  Elucidating tools, patterns, and practices
+        1. Functions as a Service
+            a. Variable cost application models
+        2. Utility computing
+        3. Frameworks
+            a. Serverless
+            b. Others?
+
+II.  Why should I use serverless?
+    A.  Helping you do what you know you should be doing already
+        1. API-driven development
+            a. Platforms as a first-class citizen
+            b. Meta-data analysis and informed decision making
+        2. Value chain analysis
+            a. Purposefully evolve the ecosystem
+            b. Focus on value creation
+        3. Two pizza teams
+            a. Decoupled architecture
```

Comment:
I've also heard 'evolutionary architecture' used as a way to describe the ability to not lock yourself in to specific tools/patterns. The GameStop example would be logging. Right now, we push data to the ELK stack, but we should be able to evolved that portion independently if they want to switch/add logging tools down the road (Splunk, log.ly, etc)